lib/template/details-json.php

```diff
@@ -26,7 +26,9 @@
 
 $aPlaceDetails['extratags'] = $aPointDetails['aExtraTags'];
 $aPlaceDetails['calculated_wikipedia'] = $aPointDetails['wikipedia'];
-$aPlaceDetails['icon'] = CONST_Website_BaseURL.'images/mapicons/'.$aPointDetails['icon'].'.n.32.png';
+if (isset($aPointDetails['icon'])) {
```

Comment:
This gets set to false in website/details.php:139, so you need to check for `if ($aPointDetails['icon'])`.

Reply:
fixed, squashed and force pushed